src/Campaigns/Repositories/CampaignRepository.php

```diff
@@ -262,6 +262,65 @@ public function delete(Campaign $campaign): bool
         return true;
     }
 
+    /**
+     * @unreleased
+     *
+     * @param Campaign[] $campaignsToMerge
+     *
+     * @throws Exception
+     */
+    public function mergeCampaigns(array $campaignsToMerge, Campaign $destinationCampaign): bool
```

Comment:
@glaubersilva Minor suggestion, I'd use a variadic parameter here:
```php
public function mergeCampaigns(Campaign $destinationCampaign, Campaign ...$campaignsToMerge)
```

Note that I am swapping the parameter order.

Reply:
@JasonTheAdams Thanks for the suggestion! Resolved in c3db75b53b83d2449c7b0cae4e2a003c7f310a32